docs/docs/aztec/concepts/pxe/index.md

```diff
@@ -8,44 +8,101 @@ importance: 1
 
 ### Database
 
-The database stores transactional data and notes within the user's PXE. In the Aztec protocol, the database is implemented as a key-value database backed by LMDB. There is an interface ([GitHub](https://github.com/AztecProtocol/aztec-packages/blob/ca8b5d9dbff8d8062dbf1cb1bd39d93a4a636e86/yarn-project/pxe/src/database/pxe_database.ts)) for this PXE database that can be implemented in other ways, such as an in-memory database that can be used for testing.
+The database stores transactional data and notes within the user's PXE.
 
 The database stores various types of data, including:
 
-- **Notes**: Encrypted representations of assets.
-- **Deferred Notes**: Notes that are intended for a user but cannot yet be decoded due to the associated contract not being present in the database. When new contracts are deployed, there may be some time before it is accessible from the PXE database. When the PXE database is updated, deferred note are decoded.
-- **Authentication Witnesses**: Data used to approve others from executing transactions on your behalf.
+- **Notes**: Data representing users' private state. These are often stored on-chain, encrypted to a user. The PXE will parse on-chain data to find notes relevant for accounts that it is aware of.
+- **Deferred Notes**: Notes that are intended for a user but cannot yet be decoded due to the associated contract not being present in the database. When new contracts are deployed, there may be some time before it is accessible from the PXE database. When the PXE database is updated, deferred notes are decoded.
+- **Authentication Witnesses**: Data used to approve others for executing transactions on your behalf. The PXE provides this data to transactions on-demand during transaction simulation via oracles.
 - **Capsules**: External data or data injected into the system via [oracles](#oracles).
+- **Address Book**: A list of expected addresses that a PXE may encrypt notes for, or received encrypted notes from. This list helps the PXE reduce the amount of work required to find notes relevant to it's registered accounts.
 
 ### Note discovery
 
-There is an open RFP for how note discovery will work on Aztec. You can find more information in the [forum](https://forum.aztec.network/t/request-for-proposals-note-discovery-protocol/2584).
+Note discovery helps solve the problem of a user parsing the chain for their encrypted notes.
+
+There is a note tagging scheme that allows users to register an expected note sender in their PXE, which tells the PXE to expect notes coming from that expected sender. This helps the PXE more efficiently discover the encrypted notes being created for it's registered accounts.
+
+If communication between transacting parties is not possible before transacting, users can use a brute force method of trial decrypting all created notes in order to determine which notes are intended for them.
```

Comment:
We do not have trial decryption any more unfortunately. There are ways to receive notes from unknown parties, but we've not yet developed them.

Reply:
ok good to know, deleting